charts/topolvm/values.yaml

```diff
@@ -7,10 +7,15 @@ image:
 
   # image.tag -- TopoLVM image tag to use.
   # @default -- `{{ .Chart.AppVersion }}`
-  tag:  # 0.8.3
+  tag:
 
   # image.pullPolicy -- TopoLVM image pullPolicy.
-  pullPolicy:  # Always
+  pullPolicy:
+
+  # image.pullSecrets -- List of imagePullSecrets.
+  pullSecrets: []
+  # - pull-secret-1
+  # - pull-secret-2
```

Comment:
```suggestion
```

I don't think this comment is very useful

Reply:
I have removed those comments